### DISCUSSION TITLE
Showing a phone number on the main referral/dashboard page

### DISCUSSION THREAD

**Discussion by AGNMM18** [CareTogether/CareTogetherCMS#235] [Ideas]
Would it work well to show the family phone number on the main referrals screen, or would it work better to leave it within the family page?

**Comment by LarsKemmann** [comment]
Another idea is to show the referral partner. Any other ideas, please list here.

**Comment by LarsKemmann** [comment]
This also requires ability to enter/create a new referral partner when opening a referral.

**Comment by LarsKemmann** [comment]
Reporting on per-referral partner metrics, display who their primary contacts are.

**Comment by LarsKemmann** [comment]
Track engagement w/ referral partners - last time we presented to their staff, ... (CRM functionality)
